pkg/kamailio/deb/buster/kamailio.service

```diff
@@ -4,18 +4,16 @@ Wants=network-online.target
 After=network-online.target
 
 [Service]
-Type=forking
+Type=simple
+User=kamailio
```

Comment:
Hi!, thanks for this pull request.. finally after many fights and ignored mails, but now a small request (if it can be done): there's a way to pass the user and group from environment or invoke command? i mean for packaging.. in debian exits the /etc/default/kamailio file that set user and group...

Reply:
To change username at runtime need place drop-in into `/run/systemd/system/kamailio.service.d` directory and apply changes by `systemctl daemon-reload`
For dynamic unit updates you cannot use `/etc/systemd/system/kamailio.service.d` directory.

Also you can use [service templates](https://fedoramagazine.org/systemd-template-unit-files/)

Reply:
`there's a way to pass the user and group from environment or invoke command?`
Please look d3cc4e4c10820e538e7cc73b839140e38df09756
kamailio default have complete description how to change user/group of kamailio process.